README.md

```diff
@@ -1,17 +1,20 @@
 # Homebrew webpage
 
-The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for the index is at layouts/index.html.
+The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for
+the index is at `_layouts/index.html`.
```

Comment:
You can leave it as-is. I guess my point is I don't think keeping it under 80 chars is necessary for Markdown but I'll merge this as-is.